expyfun/io/_parse.py

```diff
@@ -73,3 +99,53 @@ def read_tab(fname, group_start='trial_id', group_end='trial_ok'):
             d[key] = [(these_vals[ii], these_times[ii]) for ii in idx]
         data.append(d)
     return data
+    
+    
+    def reconstruct_tracker(fname):
+        # read in raw data
+        raw = read_tab_raw(fname)
```

Comment:
No no no -- I'm asking about why `def reconstruct_tracker` is nested in `read_tab` after the `return` statement of `read_tab`, where it can never be reached... i.e. I think you have an indentation problem